userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -120,7 +120,10 @@ bool cri_async_source::parse_cri(sinsp_container_info& container, const libsinsp
 
 	for(const auto &pair : resp_container.labels())
 	{
-		container.m_labels[pair.first] = pair.second;
+		std::string label_val = pair.second;
```

Comment:
why are we creating a copy of this here? This is a wasted copy.

Reply:
agreed. I wanted the code clarity to show it is a string object. But perhaps not worth it.